exchange.py

```diff
@@ -16,29 +16,47 @@ def __init__(self, configuration):
         self.privatekey = configuration['privatekey']
         self.publickey  = configuration['publickey']
         self.exchange   = configuration['exchange']
-
+        
+        self.DEBUG = 100 # TODO: add to configuration
+        self.api = "https://tuxexchange.com/api" # TODO: add to configuration
+        
+    def debug(self, level, header, *args):
+        if level <= self.DEBUG:
```

Comment:
extract to constant

Reply:
We pulled out the tux API to the constants file, right? This can just be replaced with the constant

Reply:
yes we did